highs/presolve/HPresolve.cpp

```diff
@@ -4680,11 +4688,12 @@ HighsModelStatus HPresolve::run(HighsPostsolveStack& postsolve_stack) {
     if (options->presolve != kHighsOffString &&
         reductionLimit < kHighsSize_tInf) {
       highsLogUser(options->log_options, HighsLogType::kInfo,
-                   "Presolve performed %" PRId64 " of %" PRId64
-                   " permitted reductions\n",
-                   postsolve_stack.numReductions(), reductionLimit);
+                   "\nPresolve: Performed %" PRId64 " of %" PRId64
+                   " permitted true reductions\n",
```

Comment:
Should it being worded differently to something like `x reductions (y from scaling)` or `x reductions and y scaling reductions`.
That would avoid using ambiguous like "true reductions".